src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -559,8 +559,8 @@
   <data name="Convert_to_raw_string" xml:space="preserve">
     <value>Convert to raw string</value>
   </data>
-  <data name="Convert_to_raw_string_no_indent" xml:space="preserve">
-    <value>Convert to raw string (no indent)</value>
+  <data name="without_leading_whitespace_may_change_semantics" xml:space="preserve">
+    <value>... without leading whitespace (may change semantics)</value>
```

Comment:
A note to the translators about the ellipses may be useful to help them understand what it is implying.

Reply:
good point!